repository/p.json

```diff
@@ -1866,6 +1866,18 @@
 				}
 			]
 		},
+		{
+			"name": "Portugol Syntax",
+			"details": "https://github.com/rlsantos/PortugolSyntax",
+			"readme": "https://github.com/rlsantos/PortugolSyntax/blob/main/README.md",
```

Comment:
Since your readme is in the default location, you don't need to link to it here (ie. please remove this line)